Resources/Prototypes/Catalog/Fills/Items/firstaidkits.yml

```diff
@@ -89,7 +89,7 @@
 - type: entity
   id: MedkitCombatFilled
```

Comment:
combat medkit sometimes spawns in the CMO's locker, so this is kinda weird

Reply:
ill change it